cidc_api/services/ingestion.py

```diff
@@ -308,8 +308,8 @@ def upload_manifest(
         send_email=True,
     )
 
-    # Publish that this trial's metadata has been updated
-    gcloud_client.publish_patient_sample_update(trial.trial_id)
+    # Publish that a manifest upload has been received
+    gcloud_client.publish_patient_sample_update(manifest_upload.trial_id)
```

Comment:
I'm lost 
```
def publish_patient_sample_update(manifest_upload_id: int):
```
vs this here 
```
publish_patient_sample_update(manifest_upload.trial_id)
```
So should that be a `trial_id` or `manifest_upload_id` ?

Reply:
typo!